src/Nethermind/Nethermind.Optimism/OptimismChainSpecEngineParameters.cs

```diff
@@ -52,7 +54,7 @@ public void ApplyToReleaseSpec(ReleaseSpec spec, long startBlock, ulong? startTi
 
         if (HoloceneTimestamp is not null)
         {
-            spec.BaseFeeCalculator = new OptimismBaseFeeCalculator(HoloceneTimestamp.Value, new DefaultBaseFeeCalculator());
+            spec.BaseFeeCalculator = new OptimismBaseFeeCalculator(HoloceneTimestamp.Value, JovianTimestamp, new DefaultBaseFeeCalculator());
```

Comment:
I'd rather have this a bit more consistent and pass both Holocene and Jovian timestamps as optionals (most likely the `HolocenteTimestamp is not null` check should be removed).